src/main.rs

```diff
@@ -745,14 +749,22 @@ fn compile(
                     .to_string();
                 let settings = settings.get(&extension);
 
-                if track_code_files.contains(&file_path) {
-                    return Err(format!(
-                        "Multiple locations point to code file {}",
-                        file_path.display()
-                    )
-                    .into());
-                } else {
-                    track_code_files.insert(file_path.clone());
+                match track_code_files.entry(file_path.clone()) {
+                    Occupied(entry) => {
+                        if sub_source_file == *entry.get() {
+                            eprintln!("  Skipping file {} (already written)", file_path.display());
```

Comment:
Shouldn't informational output go to stdout?

Reply:
Currently, all informational output is done via `eprintln!`, as I thought stdout shoulde be reserved for machine-readable output. However, there is no such output in the current version anymore (in an older version treating only a single file. there was).

Reply:
> as I thought stdout shoulde be reserved for machine-readable output.

Yes, if you application is able to produce "results" via stdout, then all human-targeted information should go to stderr. But since Yarner does not do that, the stdout/stderr split can be used for informational/error signalling instead. Both are just conventions though, so pushing everything via stderr does work as well. It just leaves some signalling potential "on the table."

Reply:
If we really add the possibility for pre/postprocessors, we wil definitely have machine-readable output.

Reply:
But that output will probably not be transferred via our stdout. If the processors are separate binaries, we will most likely spawn child processes and exchange data via _their_ stdin/stdout, but ours would stay firmly targeted at human consumption.

Reply:
So you suggest to print everything except error messages to stdout?

Reply:
Yes, I would find that reasonable.